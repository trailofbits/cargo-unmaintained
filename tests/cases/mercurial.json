[
  {
    "name": "oorandom",
    "outdated_deps": [],
    "repo_status": "Uncloneable",
    "version": "11.1.5"
  }
]
