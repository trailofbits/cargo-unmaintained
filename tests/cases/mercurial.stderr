Scanning 1 packages and their dependencies (pass --verbose for more information)
warning: failed to clone `https://hg.sr.ht/~icefox/oorandom`: [
    "fatal: repository 'https://hg.sr.ht/~icefox/oorandom/' not found/n",
]
